Packs/Palo_Alto_Networks_Enterprise_DLP/Integrations/Palo_Alto_Networks_Enterprise_DLP/Palo_Alto_Networks_Enterprise_DLP.yml

```diff
@@ -29,7 +29,6 @@ configuration:
   - EU
   - AP
   - UK
-  defaultvalue: US,EU,AP,UK
```

Comment:
what if a customer didn't configure this param because the default value fitted for him, and upgrades to your version?

Reply:
In my case it took the supported region in my DLP env (US), so I guess it will return the result for the support regions for the customers as well.